app/views/question_flow/forms/_outgoings.html.slim

```diff
@@ -1,28 +1,51 @@
 - partner ||= false
+- conditionals = FeatureFlags.enabled?(:conditional_reveals, @check.session_data)
```

Comment:
comment - maybe re-name to `conditionals_reveal_flag`

Reply:
Good idea